src/components/SearchResult.tsx

```diff
@@ -63,11 +69,20 @@ export interface SearchResultProps {
 
 class SearchResult extends React.Component<SearchResultProps & WithStyles<typeof styles>> {
     render() {
-        const { classes, iconColor, iconSrc, iconType, label, to } = this.props;
+        const { classes, iconColor, iconSrc, iconType, query, label, to } = this.props;
 
         const isAvatar = iconType === 'avatar';
         const isIcon = iconType === 'icon';
 
+        const regex = new RegExp('(' + query + ')', 'gi');
```

Comment:
`query` moet ge-escaped worden, nu gaat het niet helemaal goed als er regexp symbolen in zitten. Iets als [dit](https://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex) misschien?